include/boost/gil/histogram.hpp

```diff
@@ -0,0 +1,298 @@
+//
+// Copyright 2020 Debabrata Mandal <mandaldebabrata123@gmail.com>
+//
+// Distributed under the Boost Software License, Version 1.0
+// See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt
+//
+
+#ifndef BOOST_GIL_HISTOGRAM_HPP
+#define BOOST_GIL_HISTOGRAM_HPP
+
+#include <boost/gil/concepts/concept_check.hpp>
+#include <boost/gil/metafunctions.hpp>
+#include <boost/gil/pixel.hpp>
+#include <boost/container_hash/extensions.hpp>
+#include <boost/mp11.hpp>
+
+#include <type_traits>
+#include <unordered_map>
```

Comment:
@mloskot should the **container_hash/extensions.hpp** be shifted? It is actually quite confusing because boost::hash_value is actually found in another place at boost/functional/hash.hpp. I will look into this and report back.

Reply:
@codejaeger I meant something like this, which is groups, then folder-based and alphabetical order within groups:

```
#include <boost/gil/metafunctions.hpp>
#include <boost/gil/pixel.hpp>
#include <boost/gil/concepts/concept_check.hpp>

#include <boost/mp11.hpp>
#include <boost/container_hash/extensions.hpp>

#include <type_traits>
#include <unordered_map>
```